backend/src/main/kotlin/app/ehrenamtskarte/backend/application/webservice/EakApplicationMutationService.kt

```diff
@@ -21,15 +21,19 @@ import org.jetbrains.exposed.sql.transactions.transaction
 
 @Suppress("unused")
 class EakApplicationMutationService {
-
     @GraphQLDescription("Stores a new application for an EAK")
     fun addEakApplication(
         regionId: Int,
         application: Application,
         project: String,
-        dfe: DataFetchingEnvironment
+        dfe: DataFetchingEnvironment,
     ): DataFetcherResult<Boolean> {
-        val applicationHandler = ApplicationHandler(dfe.getContext<GraphQLContext>(), application, regionId, project)
+        val applicationHandler = ApplicationHandler(
+            dfe.getContext<GraphQLContext>(),
+            application,
+            regionId,
+            project,
+        )
```

Comment:
:wrench: With a line length of 120 chars this should still be fine? Same for quite a few other changes.

Reply:
I think the autoformatter is a bit overly enthusiastic. I corrected a few of these instances manually, but missed this one.